examples/lightgbm_tuner_simple.py

```diff
@@ -0,0 +1,52 @@
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset using LightGBM tuner.
+
+In this example, we optimize the validation accuracy of cancer detection.
```

Comment:
Suggested a minor change to align to the script body:
```suggestion
In this example, we optimize the validation log loss of cancer detection.
```

Reply:
Fixed!